src/js/vic-v2/components/VeteranIDCard.jsx

```diff
@@ -1,7 +1,135 @@
 import React from 'react';
 
-export default function VeteranIDCard() {
-  return (
-    <div style={{ paddingTop: '95px', backgroundColor: 'gray', height: '200px', width: '300px' }}></div>
-  );
+const FONT_FAMILY = 'Arial, Helvetica, sans-serif';
+// sets the aspect ratio of svg
+const VIEW_BOX = '0 0 2048 1284';
+// height of font top to bottom bounding box in the above viewbox
+const X_LARGE_FONT_SIZE = 90;
+const LARGE_FONT_SIZE = 80;
+const SMALL_FONT_SIZE = 60;
+const X_SMALL_FONT_SIZE = 55;
+
+const branchCodeLabels = {
```

Comment:
Can you put this in the vic-v2/labels.js file, and import it from there? You can also update the form config file to pull the labels from there.